From e7a5030483e039971aad59db3fc238c3051bd565 Mon Sep 17 00:00:00 2001
From: Herbert Xu <herbert@gondor.apana.org.au>
Date: Mon, 25 Feb 2019 12:49:20 +0800
Subject: [PATCH 47/53] options: Do not set commandname in procargs

We set commandname in procargs when we don't have to.  This results
in a duplicated output of arg0 when an error occurs.

Reported-by: Olivier Duclos <odc@fastmail.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
---
 src/options.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/options.c b/src/options.c
index 6f381e6..a46c23b 100644
--- a/src/options.c
+++ b/src/options.c
@@ -159,7 +159,6 @@ procargs(int argc, char **argv)
 		setinputfile(*xargv, 0);
 setarg0:
 		arg0 = *xargv++;
-		commandname = arg0;
 	}
 
 	shellparam.p = xargv;
-- 
2.25.0

